classes/class-wc-emspay-gateway.php

```diff
@@ -9,19 +9,21 @@
 class WC_Emspay_Gateway extends WC_Payment_Gateway
 {
     protected $ems;
-    protected $allowed_currencies;
+    protected $emspay_settings;
```

Comment:
maybe be better to replace `emspay` label by `ginger` as we moving to less using of bank labels in code...